subgraphs/aggregator/schema.graphql

```diff
@@ -0,0 +1,384 @@
+type Aggregator @entity {
+  " { Hard-coded to LooksRareAggregator } "
+  id: ID!
+
+  " Total number of unique users "
+  users: BigInt!
+
+  " Total number of collections traded "
+  collections: BigInt!
+
+  " Total number of transactions "
+  transactions: BigInt!
```

Comment:
I wonder if we should distinguish between number of transactions on the aggregator vs. number of unique transactions that went through the aggregator

Reply:
e.g.

One unique transaction hash with 2 Seaport orders and 3 LooksRare orders, so the unique transaction count is 1 and the total number of transactions is 5?